frontend/src/app/features/assistance-selection/assistance-selection.component.html

```diff
@@ -3,7 +3,7 @@
     <button
       class="assistance-level-button"
       mat-raised-button
-      color="primary"
+      color="accent"
```

Comment:
Did we want to leave this `accent`, or revert back to `primary`?

Reply:
I changed it back to `primary`. We used `accent` to test that it was working.